litert_models/tests/kaggle/tensorflow/mobilenet_v1_test.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2024 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+# https://www.kaggle.com/models/tensorflow/mobilenet-v1/
```

Comment:
Starting with just this one model test in this initial PR. We can port over the previous IREE tests at https://github.com/iree-org/iree/tree/main/integrations/tensorflow/test/python/iree_tfl_tests and then also import popular models from https://www.kaggle.com/models/?orderby=downloadCount&framework=tfLite.